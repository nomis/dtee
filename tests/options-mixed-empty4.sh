#!/bin/bash
set -vx
. "$(dirname "$0")"/util/common.sh

# Empty command stops all following options parsing, even if they're empty
run_test "--debug-options" "" "-o" "" "-e" "filename"
RET=$?

variables_must_eq RET 0
