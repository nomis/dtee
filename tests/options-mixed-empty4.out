-=
-=-o
-=
-=-e
-=filename
debug-options=1
